packages/@aws-cdk/aws-appsync/lib/schema-intermediate.ts

```diff
@@ -259,3 +258,88 @@ export class InputType implements IIntermediateType {
     this.definition[options.fieldName] = options.field;
   }
 }
+
+/**
+ * Properties for configuring an Enum Type
+ *
+ * @param definition - the strings that define this enum type
+ *
+ * @experimental
+ */
+export interface EnumTypeProps {
+  /**
+   * the attributes of this type
+   */
+  readonly definition: string[];
```

Comment:
is `definition` the right terminology here? I thought the enum definition is inclusive of both the enum name and the allowed values. however, this would just be the allowed values right?

Reply:
so we only need a string of values here

i.e. ["NEWHOPE", "EMPIRE", "JEDI"]

i kept the name as definition because it is consistent with all the other intermediate types